docs/intro.md

```diff
@@ -0,0 +1,99 @@
+# Learn the Basics
+
+This chapter introduces you to the basic conception of few shot learning, and the framework of
```

Comment:
Should we modify all `few shot learning` to `Few-shot learning` ?

Reply:
we use `few shot learning` and `MMFewShot` as default.

Reply:
L9

Few-shot learning aims at generalizing to new tasks based on a limited number of 
 samples using prior knowledge.